manufacturing doc edited (#26)
Zubax/Babel-Babel: docs/manufacturing.md

# Babel-Babel manufacturing

## PCB specification for all PCB
### Main PCB

> * Quantity per product/order: 1

Reviewer: The quantity specification is redundant because it is already contained in the BOM table below, so it should be removed. You can add a forward reference if desired (like "see below").

This comment also applies to the following sections.

* Layers: 4 (layer stack in manufacturing package)
* Thickness: 1.0mm
* PCB color: Red
* Silkscreen: White
* Surface Finish: LeadFree HASL-RoHS
* Outer Copper Weight: 1 oz or 1.4 Mils
* Inner Copper Weight: 0.5 oz or 0.7 mils
* Material Type: FR4-Standard Tg 130-140C
* Flying Prob Test: Fully Test
* Castellated Holes: No

### Enclosure PCB

### Part B1

>    * Quantity per product/order: 1

Reviewer: Please fix the formatting here: add newlines after headings and remove indentation.

   * Thickness: 0.8mm
   * Surface Finish: LeadFree HASL-RoHS
   * Color: Red
   * Silkscreen: White

>    * Material: Textolite

Reviewer: I don't think this requirement is meaningful as it is unnecessarily limiting. I would recommend stating simply "FR4 or similar". This comment also applies to the following sections.

   * No Copper

### Part A
   * Quantity per product/order: 2
   * Thickness: 1.6mm
   * Surface Finish: LeadFree HASL-RoHS
   * Color: Red
   * Silkscreen: None
   * Material: Textolite
   * No Copper

### Part C
   * Quantity per product/order: 1
   * Thickness: 1.2mm
   * Surface Finish: LeadFree HASL-RoHS
   * Color: Red
   * Silkscreen: None
   * Material: Textolite
   * No Copper

### Part D
   * Quantity per product/order: 1
   * Thickness: 0.8mm
   * Surface Finish: LeadFree HASL-RoHS
   * Color: Red
   * Silkscreen: None
   * Material: Textolite
   * No Copper

The following operations must be performed for Babel-Babel manufacturing:

1) SMT mounting of the main PCB.

2) Flashing both Babels and [Dronecode probe](https://github.com/Zubax/dronecode_probe) on the PCB.
This operation should be performed without installing any connectors on the PCB,
a 4-pin pogo pin connector should be used instead.

> Connectors' pinouts used for flashing can be found in the picture below.(Flashing process will be covered in a deperate doc)

Reviewer: Do we need a separate document though? The process seems simple enough, I think we're just missing a couple of details explaining how to use drwatson. Can we address this now?

# Babel-Babel manufacturing

## PCB specification for all PCB
### Main PCB

* Layers: 4 (layer stack in manufacturing package)
* Thickness: 1.0mm
* PCB color: Red
* Silkscreen: White
* Surface Finish: LeadFree HASL-RoHS
* Outer Copper Weight: 1 oz or 1.4 Mils
* Inner Copper Weight: 0.5 oz or 0.7 mils
* Material Type: FR4
* Flying Prob Test: Fully Test
* Castellated Holes: No

### Enclosure PCB

> Every parameters for enclosure PCB are same except thickness.\

Reviewer: ```suggestion
All parameters of the enclosure PCBs are the same except thickness.
```

Common parameters: 

* Surface Finish: LeadFree HASL-RoHS
* Color: Red
* Silkscreen: White
* Material: FR4
* No Copper

### Part B1

* Thickness: 0.8mm

### Part A

* Thickness: 1.6mm

### Part C

* Thickness: 1.2mm

### Part D

* Thickness: 0.8mm

> ## The following operations must be performed for Babel-Babel manufacturing:

Reviewer: ```suggestion
## Manufacturing
```

![](figures/flashing_interface.svg)

To flash Babel-Babel, the Babel part of the board has to be flashed first then the DCP part will be flashed.

> To Flash Babel follow the procedure bellow:

Reviewer: ```suggestion
To flash and test Babel, follow the procedure below:
```

* Turn off one babel by toggling the switch
* Connect 1 DCP and 1 Babel from PC
  
![](figures/flashing1.jpg)

* Connect 1 CAN port from Babel to 1 port of Babel-Babel on the side that'll be flashed
* Navigate to drwatson_babel repository and run "./drwatson_babel.py (babel port path)"
 
![](figures/flashing3.png)

* Enter provided credentials and press enter and the flashing process will start

![](figures/flashing4.png)

> ### To flash the DCP part follow [this link](https://github.com/Zubax/dronecode_probe/blob/master/MANUFACTURING.md)

Reviewer: ```suggestion
To flash the DCP part, follow [the DCP Manufacturing documentation](https://github.com/Zubax/dronecode_probe/blob/master/MANUFACTURING.md).
```

Please do not use constructs like "this link" because they lose meaning when the text is copied or printed on a non-rich medium.

Also, this should not be a header.
